plugins/fianu/src/components/annotationHelpers.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Entity } from '@backstage/catalog-model';
+
+/** @public */
+export const FIANU_BADGE_ANNOTATION = 'dexcomdev.com/fianu-badge';
```

Comment:
The value of the annotation should be related to the plugin and it does not look like dexcomdev is related.

```suggestion
export const FIANU_BADGE_ANNOTATION = 'fianu.io/badge';
```

I'm not sure what a badge represent here, is it an id of some sort? In. that case badgeId would be preferred.

Reply:
yes makes sense. I was planning on changing this.